.github/workflows/ci.yml

```
@@ -0,0 +1,15 @@
+name: ci
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+        name: Checkout
+      - uses: cachix/install-nix-action@v12
+        name: Install Nix
+      - run: nix-build release.nix --attr spectacle
```

We previously used [`scripts/fourmolu`](https://github.com/awakesecurity/spectacle/blob/main/scripts/fourmolu) to guard against merging unformatted code, I'm not sure if this would work:
```suggestion
      - run: nix-build release.nix --attr spectacle
  fourmolu:
    runs-on: ubuntu-latest
    needs: build
    steps:
      - uses: cachix/install-nix-action@v12
        name: Check
      - run: |
        sh nix-build --no-out-link --attr fourmolu ./release.nix
```

---

I think the YAML might look a little different, but I can definitely add the `fourmolu` check 👍

---

Added in 379521545891252c96e15d292899ccc4770f24ff